minecraftcommandapi/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "rules": {
```

Comment:
please add `"extends": "../tslint.json"` to get the base rules, then add only ones that are different from the main tslint.json file.